.claude/commands/team.md

```
@@ -32,71 +34,89 @@ Generate exactly `$ARGUMENTS.subagents` elite coding agent perspectives. Each ag
+**Format:** Compound names only (e.g., `AUTH_SENTINEL`, `CACHE_STRATEGIST`). No generic roles.
 
-| Agent | Persona | Specialization |
-|-------|---------|----------------|
-| 1 | ARCHITECT | System design, scalability, architectural patterns |
-| 2 | CRAFTSMAN | Clean code, maintainability, SOLID principles |
-| 3 | GUARDIAN | Security, error handling, edge cases, testing |
-| 4 | OPTIMIZER | Performance, efficiency, resource management |
-| 5 | INTEGRATOR | APIs, interfaces, system boundaries |
+**Inject per agent:**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)